src/main/kotlin/com/intershop/gradle/isml/IsmlPlugin.kt

```diff
@@ -76,14 +76,6 @@ open class IsmlPlugin : Plugin<Project> {
                 it.group = IsmlExtension.ISML_GROUP_NAME
             }
 
```

Comment:
In a case, that there is more than one isml task in a project (two different source sets for ISML), the dependencies are added twice. If you want add the dependencies after evaluation, then it is possible to do that in the correct block.